src/Pages/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx

```diff
@@ -245,11 +251,19 @@ export default function SnapshotListModal() {
                           }
                         />
                       </Td>
-                      <Td>{content_counts?.['rpm.package'] || 0}</Td>
+                      <Td>
+                        <Button
+                          onClick={() => {
+                            navigate(`${rootPath}/${uuid}/snapshots/${snap_uuid}`);
+                          }}
```

Comment:
Read my previous comment, you need to add the variant to this button as described there.

Reply:
<img width="253" alt="Screenshot 2024-02-15 at 2 10 37 PM" src="https://github.com/content-services/content-sources-frontend/assets/38083295/413b2991-26c8-4623-9df3-6f5615935ace">
You need to remember to test your changes before committing.